keras_trainer/trainer.py

```diff
@@ -196,10 +197,12 @@ def __init__(self, **options):
         # Freeze layers if contained in list
         if self.freeze_layers_list is not None:
             for layer in self.freeze_layers_list:
-                if isinstance(layer, int):
+                if isinstance(layer, int) or isinstance(layer, np.int64):
                     self.model.layers[layer].trainable = False
                 elif isinstance(layer, str):
                     self.model.get_layer(layer).trainable = False
+                else:
+                    raise ValueError("%s layer type not supported to freeze layers" % (type(layer)))
```

Comment:
Maybe it would be more informative: 
Layer type not supported, we expect an `int` giving the layer index or a `str` containing the name of the layer.